components/brave_wallet_ui/utils/datetime-utils.ts

```diff
@@ -20,7 +20,17 @@ const monthMap = [
   'Dec'
 ]
 
-export function formatDateAsRelative(date: Date, now: Date = new Date()) {
+/**
+ * usePrecision will return a precise relative time of
+ * (minutes:seconds) or (hours:minutes) instead of a
+ * single rounded value of seconds, minutes or hours.
+ * @param {boolean} usePrecision
+ */
+export function formatDateAsRelative(
+  date: Date,
+  now: Date = new Date(),
+  usePrecision?: boolean
```

Comment:
Let's move this comment to above the param
```suggestion
export function formatDateAsRelative(
  date: Date,
  now: Date = new Date(),
  /**
 * enabling `usePrecision` will return a precise relative time of
 * (minutes:seconds) or (hours:minutes) instead of a
 * single rounded value of seconds, minutes or hours.
 */
  usePrecision?: boolean
```

Reply:
Moved https://github.com/brave/brave-core/compare/598edeb9330b6cf4b323844f0e7ad090185516d0..0c201842c3a57b6f0d21c6a6d94d937008b38c4c